common/scripted_effects/hfe_effect.txt

```diff
@@ -752,18 +792,23 @@ hfe_effect_varelse_fleet_find_new_home = {
 hfe_effect_varelse_fleet_find_new_home_make_colony = {
 	from = {
 		create_country = {
-			name = "Varelse Hegemony"
+			name = "Neo-Varelse Hegemony"
 			type = default
 			species = event_target:hfe_varelse_species
+			origin = "hfe_civic_features_exodus"
 			ethos = {
 				ethic = ethic_xenophobe
 				ethic = ethic_militarist
 				ethic = ethic_materialist
 			}
 			effect = {
+				set_country_flag = hfe_was_government_in_exile
+				set_country_flag = hfe_chosen_to_exodus
 				set_country_flag = hfe_varelse
+				set_graphical_culture = fallen_empire_02
```

Comment:
Doesnt it have graphical_culture from species?

Reply:
No. You have to set it in the Country Scope.